backend/src/common/enums/bcws/skills-certifications.enum.ts

```diff
@@ -22,5 +22,4 @@ export enum CertificationName {
   VOLUNTEER_FIRE_DEPT = 'Volunteer Fire Dept',
   FORKLIFT = 'Forklift - Certified',
   PSYCHOLOGICAL_FIRST_AID = 'Psychological First Aid (PFA)',
-  RRT = 'Resource RequestTracking',
```

Comment:
Do we need to update the insert script for certs?

Reply:
Don't think so; RRT is a Tool, I'm just removing it as a Cert because it doesn't appear to be one